### PR TITLE
Clarify labels and tooltip in form creation for "Render style"

### LINKED ISSUE #8615: Clarify labels and tooltip in form creation for "Render style"
| Q   | A
| --- | ---
| Mautic version | 2.16.0 |

## Bug Description
This is not properly a bug but more a labeling issue.

Creating a new form, the option to disable rendering is confusing.

Schermata 2020-04-01 alle 18 32 23

"Render styles?": No, I don't want them to be rendered, so I select "No"
[Tooltip] "Disable CSS rendering": Mmm, so I have to select "yes" as I want to disable CSS rendering?

I suggest a better labeling like:
Label: Render styles?
Tooltip: Enable CSS rendering

This way the two labels direct the action in the same direction: "Yes" to enable CSS, "No" to disable them.

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/90857453-clarify-labels-and-tooltip-in-form-creation-for-render-style?utm_campaign=plugin&utm_content=tracker%2F5355074&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F5355074&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Aerendir**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | no
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
-->
| Issue(s) addressed                     | Fixes #8615 

This is not properly a bug but more a labelling issue.

Creating a new form, the option to disable rendering is confusing.

image

"Render styles?": No, I don't want them to be rendered, so I select "No"
[Tooltip] "Disable CSS rendering": Mmm, so I have to select "yes" as I want to disable CSS rendering?

This PR implements a better labelling like:
Label: Render styles?
Tooltip: Enable CSS rendering

This way the two labels direct the action in the same direction: "Yes" to enable CSS, "No" to disable them.

### CODE REVIEW

**Comment by RCheesley** [comment]
@Aerendir please can you sign the CLA above? Also it might be good to make a PR to the docs so that we can make it clear what this does and how it works - you can do that at https://github.com/mautic/mautic-documentation.

I have updated the title and description based on the issues but can you please add step by step how to reproduce this issue and what should be seen with the PR applied? Thanks!

**Review by npracht — COMMENTED** [review]
👍🏽

**Comment by RCheesley** [comment]
@cla-bot check

**Comment by RCheesley** [comment]
@Aerendir we need you to sign the contributors agreement in order to consider this PR for merging - the link is https://www.mautic.org/contributor-agreement and you can do it online, just use the link at the top of the page!

**Comment by Aerendir** [comment]
@RCheesley , I've just signed it!

**Comment by RCheesley** [comment]
@cla-bot check

File: app/bundles/FormBundle/Translations/en_US/messages.ini [hunk 1]
```diff
@@ -136,7 +136,7 @@ mautic.form.form.help.searchcommands="<strong>Search commands</strong><br />ids:
 mautic.form.form.kioskmode="Kiosk Mode"
 mautic.form.form.kioskmode.tooltip="If set to yes, form submissions will not generate new contact tracking cookies or assign the IP address to created/updated contacts."
 mautic.form.form.renderstyle="Render Style"
```
Comment (RCheesley) [inline app/bundles/FormBundle/Translations/en_US/messages.ini]:
```suggestion
mautic.form.form.renderstyle="Render style from Template?"
```

File: app/bundles/FormBundle/Translations/en_US/messages.ini [hunk 2]
```diff
@@ -136,7 +136,7 @@ mautic.form.form.help.searchcommands="<strong>Search commands</strong><br />ids:
...
-mautic.form.form.renderstyle.tooltip="Disable CSS rendering"
+mautic.form.form.renderstyle.tooltip="Disable/Enable rendering of CSS"
```
Comment (RCheesley) [inline app/bundles/FormBundle/Translations/en_US/messages.ini]:
```suggestion
mautic.form.form.renderstyle.tooltip="Enable/Disable rendering of CSS from Template"
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Thanks for the PR and I agree this could be clearer - I have made a small suggestion as some folk may not know what we mean when we refer to CSS. 

See what you think - happy to go with this or with your suggestion!

File: app/bundles/FormBundle/Translations/en_US/messages.ini (hunk 2 above)
Comment (Aerendir) [reply app/bundles/FormBundle/Translations/en_US/messages.ini]: I added the part "from template", but leaved the order "Disable/Enable" as on the left we have "No" and on the right we have "Enable", so "Disable/Enable" follows the order of the choices.

**Review by kuzmany — APPROVED** [review]
👍

**Review by RCheesley — APPROVED** [review]

**Pull request merged**